src/Dependencies/Installer.php

```diff
@@ -96,4 +87,22 @@ private function installDependency($type, $dep)
 
         return \Result\Result::ok();
     }
+
+    private function inspectionDetailsMessage($type, $dep)
+    {
+        $result = $this->inspectionChecker->check($type, $dep);
+
+        if (!$result->isErr()) {
+            $inspectionDetails = $result->unwrap();
+            if (!empty($inspectionDetails)) {
+                $message = sprintf("%s\n", $inspectionDetails);
```

Comment:
I don't think that works: if `empty($result->unwrap())` then you'll end up in the `else` branch....